cypress/fixtures/properties/property.json

```diff
@@ -41,5 +41,19 @@
   "tenure": {
     "typeCode": "SEC",
     "typeDescription": "Secure"
+  },
+  "contacts": {
```

Comment:
I've seen your comments to propose a change to this nesting and totally agree that this makes no sense like this

Reply:
Backend has been updated, and frontend matched up here: https://github.com/LBHackney-IT/repairs-hub-frontend/pull/89/commits/499c4f19c4bdf645eb77c6893bbee9a5e72a5bfb